src/hooks/index.js

```diff
@@ -0,0 +1,7 @@
+/** @type {import('@sveltejs/kit').Handle} */
+export async function handle({ request, resolve }) {
```

Comment:
Change required due to recent sveltekit update sveltejs/kit#2804. Not sure if this correct for running the site on production though.

Reply:
ok, I wasn't aware, thanks. Seems to work on the dev-version